bundled-libs/masterminds/html5/bin/entities.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
@garvinhicking maybe this is the file that you meant. It's a .php file that would be under bundled_libs and not bound to a class, so this one would run when called in a browser, if not otherwise protected